src/main/java/duke/Launcher.java

```diff
@@ -0,0 +1,13 @@
+package duke;
+import duke.gui.Main;
+import javafx.application.Application;
```

Comment:
I normally add a empty line between each imports from different source to make it tidier